app/models/repeatable_step.rb

```diff
@@ -0,0 +1,78 @@
+class RepeatableStep < Step
+  include ActionView::Helpers::SanitizeHelper
+  include ActionView::Helpers::TagHelper
+  include ActionView::Helpers::OutputSafetyHelper
+
+  attr_accessor :answer_id, :questions
```

Comment:
Do we want to rename this to `index`?

```suggestion
  attr_accessor :answer_index, :questions
```

Reply:
yes, I think that makes sense.